assignments/callbacks.js

```diff
@@ -1,46 +1,35 @@
-// Create a callback function and invoke the function to test your work. You have been provided an example of a problem and a solution to see how this works with our items array.  Study both the problem and the solution to figure out the rest of the problems.
-
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
-/* 
-
-  //Given this problem: 
-  
-  function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
-  }
-
-  // Potential Solution:
-  function firstItem(arr, cb) {
-    return cb(arr[0]);
-  }
-
-  firstItem(items, function(first) {
-    console.log(first)
-  });
-
-*/
-
+function firstItem(arr, cb) {
+  cb = arr[0];
+  return cb;
+}
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  cb = items.length;
+  return cb;
```

Comment:
compare to solution code:
function getLength(arr, cb) {
  console.log(cb(arr));
}